src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -128,11 +136,14 @@ public void feederMotorStop() {
     }
 
     /**
-     * Checks if ball is positioned at the second sensor
+     * Checks if ball is positioned at the sensors
      */
     @Log(name = "Has Cargo")
     public boolean hasCargo() { // might rename methods later?
-        return feederProximity.get();
+        if (ignoreFeeder) {
+            return feederOverridenValue;
+        }
+        return !(leftFeederProximity.get() && rightFeederProximity.get());
```

Comment:
I don't think the sensors need to be inverted, especially since the previous logic didn't invert them and seemed to work fine when the sensors did.
(The fact that erroneous `true` values are possible just means that `&&` is correct instead of `||`, it doesn't affect any inversion of the values.)